py/trtorch/logging.py

```diff
@@ -26,6 +27,8 @@ def _to_internal_level(external) -> LogLevel:
             return LogLevel.INFO
         if external == Level.Debug:
             return LogLevel.DEBUG
+        if external == Level.Graph:
+            return LogLevel.GRAPH
```

Comment:
We should probably add an exception or something here if the program gets this far into the cases.